### ISSUE TITLE
CI fully covered for 4.0 major release

### ISSUE THREAD

**Issue by lifupan** [kata-containers/kata-containers#9542]
**Which feature do you think can be improved?**

Specify the feature you think could be made better.

**How can it be improved?**

Describe how specifically you think it could be improved.

**Additional Information**

Anything else to add?

**Before raising this enhancement request**

Have you looked at the [limitations document](https://github.com/kata-containers/kata-containers/blob/main/docs/Limitations.md)?

**Kata Containers survey**

Please consider taking the survey to help us help you: https://openinfrafoundation.formstack.com/forms/kata_containers_user_survey

**Comment by stevenhorsman** [comment]
@lifupan @Apokleos - do you think we can close this issue out now? I think we have CI coverage for the core features for 4.0 on runtime-rs?

**Comment by lifupan** [comment]
Yes, as it stands now, the CI of runtime-rs has been fully aligned with the Go runtime. We can close this issue.

**Comment by Apokleos** [comment]
Detailed feature parity, please check https://github.com/kata-containers/kata-containers/issues/8702
